src/Cart.tsx

```diff
@@ -0,0 +1,196 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable max-len */
+import { NavLink } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { NotFoundItems } from './NotFound';
+import { useAppSelector } from './utils/hooks';
+import { addItem, minusItem, removeItem } from './redux/cartReducer';
+
+export const Cart = () => {
+  const { items } = useAppSelector((state) => state.cart);
+  const dispatch = useDispatch();
+
+  const sum = items.reduce((prev, current) => prev + current.price, 0);
+
+  const baseUrl = 'https://mate-academy.github.io/react_phone-catalog/_new/';
+  const counts = items.map((item) => item.count);
+
+  const calculateTotalForAllItems = () => {
+    let totalToBasket = 0;
+
+    for (let i = 0; i < items.length; i++) {
+      totalToBasket += items[i].price * counts[i];
+    }
+
+    return totalToBasket;
+  };
+
+  const calculateTotalItems = () => {
+    let totalСounts = 0;
+
+    for (let i = 0; i < counts.length; i++) {
+      totalСounts += counts[i];
+    }
+
+    return totalСounts;
+  };
+
+  useEffect(() => {
+    localStorage.setItem('sumToSave', sum.toString());
+  }, [items]);
+
+  const handleAdd = (index: number) => {
+    const selectedPhone = items[index];
+
+    dispatch(addItem(selectedPhone));
+  };
+
+  const handleRemove = (index: number) => {
+    const selectedPhone = items[index];
+
+    if (selectedPhone.count > 1) {
+      dispatch(minusItem(selectedPhone));
+    } else {
+      dispatch(removeItem(selectedPhone));
+    }
+  };
+
+  return (
+    items.length > 0 ? (
+      <div className="basket">
+        <div className="filter__nav">
+          <div className="filter__nav--1-item">
+            <NavLink to="#phones" className="page__link">
+              Back
+            </NavLink>
```

Comment:
what if I will open the Cart.tsx component from the home page and press the Back button?
it will lead me to phones instead of the home page, let's fix that